app/controllers/search_controller.rb

```diff
@@ -3,8 +3,8 @@ class SearchController < ApplicationController
   def search
     @name = params[:summoner]
     if valid_search?
```

Comment:
Found a behavior that is closely related to the valid_search function and was introduced in the search_refactor merge but can be fixed here as well. On this current branch, when you try to invite a LoL player that is not on LeagueSpec, you are never able to get the invitation done successfully because the code used in search_controller's invite function calls the downcase_and_search function. The downcase_and_search function however no longer exists. It was in LolConnection initially but was changed to valid_search? and moved to a different file. To test this, try inviting a user using code from this pull request and compare searching the same user using code from the invitation_functionality merge.